### ISSUE TITLE
Not clear "Usage" instruction

### ISSUE THREAD

**Issue by AndreySurzhan** [sebdah/git-pylint-commit-hook#60]
Hi all,
I would like to sorry in advance if it is just me not able to understand the instruction.
I tried several things and it seems `git-pylint-commit-hook` is just not triggered on `git commit`

*My settings:*

- `git-pylint-commit-hook==2.2.2`
- `Python 3.6.0 (v3.6.0:41df79263a11, Dec 23 2016, 07:18:10) [MSC v.1900 32 bit (Intel)] on win32`
- no custom pylint configs

**Test case#1**
I tried to follow that instruction:
[README.md](https://github.com/sebdah/git-pylint-commit-hook/blob/master/README.md)
When I do commit hook is not triggered and commit is created successfully

**Test case#2**
I tried to follow that instruction:
http://git-pylint-commit-hook.readthedocs.io/en/latest/usage.html#usage
When I do commit I see the following
`.git/hooks/pre-commit: line 2: git-pylint-commit-hook: command not found`

Please let know what I am doing wrong

**Comment by volviq** [comment]
'.git/hooks/pre-commit: line 2: git-pylint-commit-hook: command not found'
If you execute 'git-pylint-commit-hook' and 'which git-pylint-commit-hook' - what does it tell you?

**Comment by AndreySurzhan** [comment]
@volviq 
Unfortunately I am developing under Windows :) so `which` doesn't exist
but `where git-pylint-commit-hook`  shows me the following:
`%project_root_path%\venv\Scripts\git-pylint-commit-hook`

**Comment by AndreySurzhan** [comment]
```
(venv) %project_root%>git-pylint-commit-hook
'git-pylint-commit-hook' is not recognized as an internal or external command,
operable program or batch file.
```

**Comment by AndreySurzhan** [comment]
I haven't tested that under Unix (mac or linux)

**Comment by volviq** [comment]
So if you execute git-pylint-commit-hook on your normal Windows shell, would it then pick up the binary? If not, you have to possibly add it to your path. I wouldn't even have expected, that this works on Windows at all.

**Comment by AndreySurzhan** [comment]
@volviq thanks for your response.
I've been using [pre-commit lib](https://pre-commit.com/) with pylint hook
That is workaround for me.
I thought that repo's being maintained that is why I didn't bother to fixing that in the first place.
If I have time I will try to fix that issue and do PR.
